i18n/es/docusaurus-plugin-content-docs/current/intermediate/zero_shot_cot.md

```diff
@@ -2,8 +2,7 @@
 sidebar_position: 4
 ---
 
-# 🟢 Zero Shot Chain of Thought
-
+# 🟢 Cadena de pensamiento sin entrenamiento previo
```

Comment:
Why is `sin entrenamiento previo` necessary?

Reply:
Context. Text below the H1 is "La cadena de pensamiento sin entrenamiento previo (Zero Shot Chain of Thought, Zero-shot-CoT"